core/src/repair_service.rs

```diff
@@ -387,7 +387,7 @@ impl RepairService {
                 info!("repair_stats: {:?}", slot_to_count);
                 if repair_total > 0 {
                     datapoint_info!(
-                        "serve_repair-repair",
+                        "repair_service-my_requests",
```

Comment:
I do understand the original names are ambiguous,  but these renames (and the other ones below) will break existing dashboards/queries. cc @c3pko 
Also until all the cluster upgrades to the new code we will have version skew where half of the nodes will submit to one table and the other half to the other table.

Reply:
yeah I hate these metric names where I'm willing to eat the pain of switching.

I suspect in most cases we'll be looking at single validators/leaders anyways